provider/aws/s3/bucket.go

```diff
@@ -16,6 +16,10 @@ type Bucket struct {
 func (b *Bucket) HasPublicExposureACL() bool {
 	for _, publicACL := range []string{"public-read", "public-read-write", "website", "authenticated-read"} {
 		if b.ACL.EqualTo(publicACL) {
+			// if there is a public access block, check the public ACL blocks
+			if b.PublicAccessBlock.IsManaged() {
```

Comment:
Can PublicAccessBlock be `<nil>` here? Can't remember if it's a pointer or not.

Reply:
the broken tests suggest yes :grimacing:

Reply:
Fixed @liamg